src/patternfly/components/Alert/alert-link.hbs

```diff
@@ -0,0 +1,6 @@
+<a class="pf-c-alert-link{{#if alert-link--modifier}} {{alert-link--modifier}}{{/if}}"
```

Comment:
this shouldn't have the `.pf-c-alert-link` class unless it's an element of the Alert component. And if you wanted to make it an element, call it `.pf-c-alert__link` and add it to the Alert docs. I left  [comment](https://github.com/patternfly/patternfly-next/pull/735#discussion_r221391914) in the main discussion thread about it.